### ISSUE TITLE
few step faster_rcnn_inception_v2_coco_2018_01_28

### ISSUE THREAD

**Issue by leccyril** [tensorflow/models#4634]
I use this model to detect as Samsung Gear S3 , i annotated almost 350 pictures, 280 in train and rest in test, 

i launch training script bit after 442 steps (with loss graphical 0.02) it increase to 25... and when i test the detection with this training even i test on a apple watch it is detected as a samsung watch. Why only fews steps and then out of graphical it increase instead of decreasing ? i know it must have like 10000 steps why i have so few and moreover it doe'snt work ?

**Comment by leccyril** [comment]
ow sorry,

Windows 10
tensorflow install from python 3 (v1.8)
icore 7 16go memory
bazel ( i don't know ? )
CUDA/cuDNN i don't know the meaning

i use this example [tutorial
[tosend.zip](https://github.com/tensorflow/models/files/2139941/tosend.zip)

](https://github.com/EdjeElectronics)

i am sorry but i am novice.

what i am not understand, why only 500 step an loss very low ?
why when i scan an image with a apple watch, it detect samsung ?

my case 

i have lot of objects to detect with brand and model (iphone X , s9, smartwacth samsung or apple).

Then i will scan picture, could be nothing into to detect, should be only iphone, could be only s9, maybe both.

First i am trying with only one object. i train almost 350 picture 300 train 50 test

Is evaluate script is important ? 

when there is for example in one picture apple and S9 have i to annotate the apple ? if y want only detect the s9 ? 

if i want detect both, better to annotate all in same picture when it is ? 

thank you for your great help !

**Comment by leccyril** [comment]
I see if i train with ssd_inception an 700 steps, it works... Maybe faster rcnn doesnt Ork with few step and only 1 class

**Comment by k-w-w** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
